src/titiler/application/titiler/application/main.py

```diff
@@ -15,18 +16,37 @@
 )
 from titiler.mosaic.errors import MOSAIC_STATUS_CODES
 
-from fastapi import FastAPI
+from fastapi import APIRouter, FastAPI
 
 from starlette.middleware.cors import CORSMiddleware
 from starlette.requests import Request
 from starlette.responses import HTMLResponse
 from starlette_cramjam.middleware import CompressionMiddleware
 
+LEVELS = {
+    "debug": logging.DEBUG,
+    "info": logging.INFO,
+    "warning": logging.WARNING,
+    "error": logging.ERROR,
+    "critical": logging.CRITICAL,
+}
+# Remove any AWS-injected logger handlers to fix Lambda logging to CloudWatch
+# https://stackoverflow.com/a/45624044
+base = logging.getLogger()
+if base.handlers:
+    for handler in base.handlers:
+        base.removeHandler(handler)
+logging.basicConfig(level=LEVELS.get(os.environ.get("LOGLEVEL", "info")))
```

Comment:
Some boilerplate we often use in Python Lambdas that:
- Allows us to configure the log level via environment variable
- Fixes a Lambda logging issue